src/component.js

```diff
@@ -53,6 +53,9 @@ function prerenderTemplate(opts) {
 function containerTemplate(opts) {
     _set_default_props(opts);
 
+    opts.frame.title = 'SeamlessAccess Button'
+    opts.frame.ariaLabel = 'SeamlessAccess Button'
```

Comment:
This fixes the accessibility issue in the IFrame. I scanned through Zoid documentation and couldn't find an answer, when I looked at the "opts" object I saw that "opts.frame" has all of the Iframe settings.  While this is the right solution, I don't think this is the best place to set those options. I couldn't find where the "opts" were actually set, maybe they aren't and we're just using default value here.